packages/@react-aria/color/src/useColorArea.ts

```diff
@@ -413,10 +410,6 @@ export function useColorArea(props: AriaColorAreaOptions, state: ColorAreaState)
       ...colorAreaStyleProps,
       role: 'group'
     },
-    gradientProps: {
-      ...gradientStyleProps,
-      role: 'presentation'
-    },
```

Comment:
Removed the need for an additional gradient element inside the color area. Now both gradients are applied to the root color area element.